extension/source/types/window.d.ts

```diff
@@ -0,0 +1,10 @@
+import type QuillController from '../Controllers/QuillController';
+import type KeyringController from '../Controllers/Keyring/KeyringController';
+
+export declare global {
+  interface Window {
+    // TODO (merge-ok) Remove both of these when full controller linking is completed.
+    QuillController: () => QuillController;
+    KeyringController: () => KeyringController;
+  }
+}
```

Comment:
@voltrevo This should address typing issues, and give us an easy point to remove the ad-hoc injection once we get more hooked up.